test/unit/test_config.py

```
@@ -38,7 +38,7 @@ def test_correct_config_defaults(monkeypatch):
     assert cfg.ctx_size == 0
     assert cfg.engine in ["podman", "docker", None]
     assert cfg.env == []
-    assert cfg.host == "0.0.0.0"
+    assert cfg.host in ["::", "0.0.0.0"]
```

Ruff (0.15.9)

[error] Possible binding to all interfaces

(S104)